Project 2/four-card-feature/style.css

```diff
@@ -0,0 +1,104 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+
+body {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	height: 100vh;
```

Comment:
to jest mocno ograniczające. Gdyby np. na mobilnym designie elementy byłyby trochę dłuższe, to kontent wychodziłby Ci poza body :)